src/Microsoft.AspNetCore.Http.Abstractions/HostString.cs

```diff
@@ -25,6 +26,31 @@ public HostString(string value)
         }
 
         /// <summary>
+        /// Creates a new HostString without modification.
+        /// </summary>
+        /// <param name="value">The value should be Unicode rather than punycode. IPv6 addresses must use square braces.</param>
```

Comment:
Should I use host instead then (based on another comment stating it's not domain but host actually)